installer/scripts/tue-install-impl.bash

```diff
@@ -467,6 +468,20 @@ function tue-install-snap
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
+function tue-install-dpkg
```

Comment:
I would also like to see an implementation in the yaml parser. Or is there a reason why this is not done (at nobleo)?

Reply:
That makes sense, I'll can add this if desired/
Here we use it in some internal script.

Reply:
Yep that would be nice